src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/PersonPicture.xaml

```diff
@@ -74,9 +77,9 @@
                         </VisualStateManager.VisualStateGroups>
 
                         <Ellipse
-                            Fill="{ThemeResource PersonPictureEllipseFillThemeBrush}"
-                            Stroke="{ThemeResource PersonPictureEllipseFillStrokeBrush}"
-                            StrokeThickness="{ThemeResource PersonPictureEllipseStrokeThickness}"
+                            Fill="{TemplateBinding Background}"
+                            Stroke="{TemplateBinding BorderBrush}"
+                            StrokeThickness="{TemplateBinding BorderThickness}"
```

Comment:
Should we also update https://github.com/unoplatform/uno/blob/master/src/Uno.UI.FluentTheme.v1/Resources/Version1/PriorityDefault/PersonPicture.xaml ?

Reply:
👍 I have updated version 1 accordingly.